### ISSUE TITLE
Cannot add Youtube Music

### ISSUE THREAD

**Issue by Frutzelkopf** [pascalopitz/unoffical-sonos-controller-for-linux#203]
First of all: Great stuff!
Im a bloody starter with Raspberry Pi 4 and Raspberry Os...
I managed to install this wonderful program, but i cant add Youtube Music and Apple Music...
Can anybody help? Apple wont work?
Greetings and thank you !

**Comment by MueKo69** [comment]
I agree with Frutzelkopf. I can't add YouTube Music to the Sonos controller. When I click on Next, nothing happens.
